packages/router/src/lib/operations/auction.ts

```diff
@@ -234,34 +234,40 @@ export const newAuction = async (
     });
   }
 
-  // No gasfees status check required for routerContractAddress
-  if (!isRouterContract) {
-    const [senderBalance, receiverBalance] = await Promise.all([
-      txService.getBalance(sendingChainId, routerAddress),
-      txService.getBalance(receivingChainId, routerAddress),
-    ]);
-    logger.debug("Got balances", requestContext, methodContext, {
-      senderBalance: senderBalance.toString(),
-      receiverBalance: receiverBalance.toString(),
-    });
+  // Check for minimum gas on both chains
+  // TODO: check for proper values for different assets
+  const LOW_GAS = parseEther("0.1");
```

Comment:
Shouldn't this be taken from the config? https://github.com/connext/nxtp/blob/main/packages/router/src/config.ts#L347

Reply:
Never mind, this is just for logs.